src/content/docs/style-guide/writer-workflow/github-intro.mdx

```diff
@@ -140,6 +135,25 @@ The Hero currently merges three times a day: At 9 AM, noon, and 3 PM Pacific.
 * Slackbot will remind us about this in #docs_deploys.
 * The hero (or delegate) is the one who should create a PR for this and merge it.
 
+## Github labels [#labels]
```

Comment:
I'd place this section right after Track issues section, before Deal with references. But up to you! Looks great in any case!